### PR TITLE
R_datasets

### PR DESCRIPTION
**Pull request by 1009205918**
This cheatsheet summarized some R built-in data sets to help people find their interested data sets to play with. For each data set, this cheatsheet listed most attributes of it with some brief explanations.

### CODE REVIEW

File: R_datasets.Rmd [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# R data sets
```
Comment (alexnsaunders) [inline R_datasets.Rmd:1]: File name should only have lower case letters, i.e. r_datasets.Rmd - otherwise they don't sort nicely.

**Review by alexnsaunders — CHANGES_REQUESTED** [review]

File: R_datasets.Rmd (hunk 1 above)
Comment (1009205918) [reply R_datasets.Rmd]: already changed

**Review by alexnsaunders — APPROVED** [review]

**Comment by alexnsaunders** [comment]
You can view your chapter here - congrats!
https://jtr13.github.io/cc21fall2/r-data-sets.html

If you look at https://jtr13.github.io/cc21fall2/github-submission-instructions.html 
in 3.3.6 you can see how you can link your file more directly to resources folder. 

[OPTIONAL] If you have other resources (such as images) included in your project, create a folder under resources/. In our example, it is resources/sample_project/. Put the resources files there. Be sure to change all the links in your .Rmd to include resources/.../, for example:

Test Photo

**Comment by alexnsaunders** [comment]
https://jtr13.github.io/cc21fall2/r-data-sets.html Here is your chapter congrats!!

**Pull request merged**